vendor_projects.json

```diff
@@ -32,7 +32,7 @@
     "source_package":{
       "download_cmd":"curl",
       "url":"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.25.0/codemirror.min.js",
-      "option":"-o",
+      "option":["-sSL", "-o"],
```

Comment:
The "-o" option should not be here. The `-o` means to write the output to a named file. The logic should be this: if a custom `name` is given (which usually is for every case), then the Python could should concatenate  `["-o", name_var]` to the argument list.

Reply:
Yes, we could put it there but the `-o` is needed even if no custom name is used.
```
       -o, --output <file>
              Write output to <file> instead of stdout.
```

Reply:
I think @whisperity is right. Put `-o` inside the python code.